test/format/backup.c

```diff
@@ -580,6 +609,13 @@ backup(void *arg)
         /* If we're taking a full backup, create the backup directories. */
         if (full || incremental == 0) {
             testutil_create_backup_directory(g.home);
+
+            /*
+             * Copy format-specific files into the backup directories so that test/format can be run
+             * on the BACKUP.copy database for verification. These include CONFIG and any
```

Comment:
Minor suggestion: I would remove the last sentence: "These include..." from here and add it to the comment at the top of the `copy_format_files` instead. That is an implementation detail of `copy_format_files` and could change sometime. The details should be with the function. Here we only care that the function copies whatever is necessary.